Updating css-loader to 2.x

damianpm opened the pull request:
**What this PR does / why we need it**:

Updating `css-loader` to 2.x

Note: a previous [PR](https://github.com/3scale/porta/pull/1151) was reverted because master was failing. Trying again now that master issues are fixed

**Which issue(s) this PR fixes** 

https://issues.jboss.org/browse/THREESCALE-3303

Martouta commented:
@damianpm @josemigallas I don't understand why this is ok but https://github.com/3scale/porta/pull/1117 had to be reverted in https://github.com/3scale/porta/pull/1151
I just trust you though 🤷‍♀

damianpm commented:
> @damianpm @josemigallas I don't understand why this is ok but #1117 had to be reverted in #1151
> I just trust you though 🤷‍♀

@Martouta Because master was failing wit an unknown npm error, so I reverted several PR's. I think the issue was flow, fixed in https://github.com/3scale/porta/pull/1153.
I hope this time we have no issues :)
